docs/data-sources/aws_privatelink.md

```diff
@@ -3,19 +3,19 @@
 page_title: "aiven_aws_privatelink Data Source - terraform-provider-aiven"
 subcategory: ""
 description: |-
-  The AWS Privatelink resource allows the creation and management of Aiven AWS Privatelink for a services.
+  Gets information about an AWS PrivateLink connection for an Aiven service.
 ---
 
 # aiven_aws_privatelink (Data Source)
 
-The AWS Privatelink resource allows the creation and management of Aiven AWS Privatelink for a services.
+Gets information about an AWS PrivateLink connection for an Aiven service.
 
 ## Example Usage
 
 ```terraform
-data "aiven_aws_privatelink" "foo" {
-  project      = data.aiven_project.foo.project
-  service_name = aiven_kafka.bar.service_name
+data "aiven_aws_privatelink" "main" {
+  project      = aiven_project.example_project.project
```

Comment:
If we use `data.aiven_project` everywhere I would stick to it. `data.aiven_project` is a datasource, so it is different from `aiven_project` (a resources). Datasources are not manageable, hence safe to use as read-only.

Reply:
Ah, okay, that makes sense. Would it be better to use data sources in general then; for example, for `aiven_kafka.example_kafka.service_name` here?

Reply:
Well, it just we use often project as a datasource. Services are rather managed by TF.

Reply:
Gotcha. I've updated here and will make sure the rest of the examples are aligned with this later. 👍